### PR TITLE
fix some major issues

### PR DESCRIPTION
**Pull request by mohi7solanki**
**Summary of changes**

- fix excess arguments in format string.

- fix dangerous mutable argument.

- fix exception message in ValueError.

To add these as checks, activate analysis on DeepSource - https://deepsource.io/signup after merging this PR. Documentation is here - https://deepsource.io/docs

The tool is free to use for open-source. Thanks.

### CODE REVIEW

File: splash/browser_tab.py [hunk 1]
```diff
@@ -687,7 +687,7 @@ def _on_javascript_window_object_cleared(self):
             try:
                 self.runjs(script, handle_errors=False)
             except Exception as e:
-                msg = "Error in autoload script #{}:".format(idx, e)
+                msg = "Error in autoload script #{}:".format(e)
```
Comment (Gallaecio) [inline splash/browser_tab.py]:
```suggestion
                msg = "Error in autoload script #{}: {}".format(idx, e)
```

**Comment by mohi7solanki** [comment]
Sure, I have removed `.deepsource.toml` and committed the suggested change, Thanks.

File: splash/browser_tab.py [hunk 2]
```diff
@@ -687,7 +687,7 @@ def _on_javascript_window_object_cleared(self):
             try:
                 self.runjs(script, handle_errors=False)
             except Exception as e:
-                msg = "Error in autoload script #{}:".format(idx, e)
+                msg = "Error in autoload script #{}:{}".format(idx, e)
```
Comment (Gallaecio) [inline splash/browser_tab.py]:
```suggestion
                msg = "Error in autoload script #{}: {}".format(idx, e)
```

**Comment by kmike** [comment]
Thanks @mohi7solanki for the fix and @Gallaecio for the review!

**Pull request merged**